internal/testutil/assert.go

```diff
@@ -0,0 +1,131 @@
+package testutil
```

Comment:
I would rename the package to assert abd move the file in internal/testutil/assert


```suggestion
package assert
```

Then remove Assert prefix from all methods

This way it would look like this in _test.go files

```diff
-     testutil.AssertNoError(t, err)
+     assert.NoError(t, err)
```

Reply:
the file is already named `internal/testutil/assert.go`

should I just change the package name then or did I misunderstand your suggestion?

Reply:
:+1:  Got it, thanks. Let me do that